electronics/sections/collision.md

```diff
@@ -69,10 +69,36 @@ Now that you know how to make a switch work as a digital input, try assembling s
 
 You’ll find some switches, prototyping board, resistors and wire in the lab.
 
+## Switch bounce
+
+ You might find that your Switch detection code is quite unreliable and seems to register lots of switching events each time you press the switch. This is because the contacts will usually 'bounce' when they are connected. This effect happens very quickly so you won't be able to see it, but your Arduino can read it's input pins fast enough for it to register each time a contact is made. To make your system more reliable you need to make sure it only triggers once each time.
```

Comment:
```suggestion
 You might find that your Switch detection code is quite unreliable and seems to register lots of switching events each time you press the switch. This is because the contacts will usually 'bounce' when they are connected. This effect happens very quickly so you won't be able to see it, but your Arduino can read its input pins fast enough for it to register each time a contact is made. To make your system more reliable you need to make sure it only triggers once each time.
```